extensions/amp-carousel/0.2/test-e2e/helpers.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const CAROUSEL_SLIDE_CLASS =
+  '.i-amphtml-element.i-amphtml-layout-responsive' +
+  '.i-amphtml-layout-size-defined.i-amphtml-built.' +
+  'amp-carousel-slide.amp-scrollable-carousel-slide' +
+  '.i-amphtml-layout';
```

Comment:
1. Can this CSS be shortened? 
2. (nit) If not, can you please move the trailing dot on line 19 to start of line 20?

Reply:
Yeah, sadly I've had no luck making it shorter. It seems like it needs the concatenation of all classes to be found and there is no `id` to search by either.